pkg/virt-api/webhooks/mutating-webhook/mutating-webhook.go

```diff
@@ -148,6 +148,61 @@ func mutateVMIs(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
 
 }
 
+func mutateMigrationCreate(ar *v1beta1.AdmissionReview) *v1beta1.AdmissionResponse {
+
+	if ar.Request.Resource != webhooks.MigrationGroupVersionResource {
+		err := fmt.Errorf("expect resource to be '%s'", webhooks.MigrationGroupVersionResource.Resource)
+		return webhooks.ToAdmissionResponseError(err)
+	}
+
+	if resp := webhooks.ValidateSchema(v1.VirtualMachineInstanceMigrationGroupVersionKind, ar.Request.Object.Raw); resp != nil {
+		return resp
+	}
+
+	raw := ar.Request.Object.Raw
+	migration := v1.VirtualMachineInstanceMigration{}
+
+	err := json.Unmarshal(raw, &migration)
+	if err != nil {
+		return webhooks.ToAdmissionResponseError(err)
+	}
+
+	// Add foreground finalizer
```

Comment:
I don't think that you mean the foreground finalizer here.

Reply:
nope :)